drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-kogito-client/src/main/java/org/drools/workbench/screens/scenariosimulation/kogito/client/editor/ScenarioSimulationEditorKogitoWrapper.java

```diff
@@ -362,32 +383,45 @@ protected void onBackgroundTabSelected() {
         scenarioSimulationEditorPresenter.onBackgroundTabSelected();
     }
 
-    protected void newFile(Path path) {
-        Command createCommand = () -> {
-            final ScenarioSimulationModel.Type selectedType = scenarioKogitoCreationPopupPresenter.getSelectedType();
+    /**
```

Comment:
It is not a part of this PR, but since you do a refactoring - probably there is some sence to change 
 protected void getModelSuccessCallbackMethod(ScenarioSimulationModel model)  -method name since it does not return anything on  setModelSuccessCallbackMethod(ScenarioSimulationModel model) ?

Reply:
@dupliaka I changed it in `onModelSuccessCallbackMethod()' because occurs in case the SCESIM model was correctly un-marshalled.